data/presets/fields/healthcare/speciality.json

```diff
@@ -4,5 +4,5 @@
     "reference": {
         "key": "healthcare"
     },
-    "label": "Specialties"
+    "label": "Specialities"
```

Comment:
I assume because it's a label this is a safe change?

Reply:
Label changes are okay if needed, but `Specialties` is the common American spelling for this and is intentional.

Reply:
Oops, fixed in Codespell in https://github.com/codespell-project/codespell/commit/ed89a55679c806759b33b88018e40e0023ecb6be .

Okay undone @quincylvania , although it's rather mixed messages being in a British English file with a US English Label, or is that covered by further translation?

Reply:
So British English is the default for OSM's tags, while US English is the default for iD's interface. This causes lots of these types of mismatches in the preset files. Kerb/curb, groyne/groin, centre/center, etc.

Reply:
Oh dear, that's entertaining!